sdk/streamlib-python-wheel/tests/single_processor_under_test.py

```
@@ -9,7 +9,9 @@ class declared inside a pytest module would have the child import the test
 suite.
 """
 
-from streamlib import RuntimeContextLimitedAccess, input, output, processor
+import numpy
+
+from streamlib import AudioBlock, RuntimeContextLimitedAccess, input, output, processor
```

Ruff (0.16.2)

[error] Import `input` is shadowing a Python builtin

(A004)